content/pt-br/docs/reference/glossary/kubeadm.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Kubeadm
+id: kubeadm
+date: 2018-04-12
+full_link: /docs/admin/kubeadm/
+short_description: >
+  Uma ferramenta para instalar rapidamente o Kubernetes e configurar um cluster seguro.
+
+aka: 
+tags:
+- tool
+- operation
+---
+ Uma ferramenta para instalar rapidamente o Kubernetes e configurar um cluster seguro.
+
+<!--more--> 
+
+Você pode usar o kubeadm para instalar a camada de gerenciamento e os componentes {{< glossary_tooltip text="nós de processamento" term_id="node" >}}.
```

Comment:
I know that the original doc does not have a glossary_tip for control plane, what do you think about adding it here?

Reply:
Can I add? I will do it. Thank you.